site/themes/multi_course/layouts/home.html

```diff
@@ -4,11 +4,45 @@
     <div id="home-search-box" class="w-100">
     </div>
   </div>
+  <div class="new-courses d-flex flex-column align-items-center">
+    {{ $pages := .Site.Pages.ByPublishDate.Reverse }}
+    {{ $coursePages := where $pages "Type" "==" "course" }}
+    {{ $childPages := where $coursePages ".Parent.Type" "==" "courseindex" }}
```

Comment:
I think here you may also be able to check if `layout == "course_home"` - I was just thinking that may be more durable in the long term since the `courseindex` page is more or less a placeholder for use during development that I think we're going to get rid of at some point.

@gumaerc do you have any thoughts about that?

Reply:
Is `layout` a field on the parent?

Reply:
it's a field which is set on each page. we have a layout (`course_home`) which is only used for the course home pages

Reply:
(would be good for @gumaerc to confirm that's true!)

Reply:
Looks like it works. The count for `.Parent.Type == courseindex` matches `.Params.layout == course_home`

Reply:
@aliceriot I think we may have to put a limit on the list of instructors and topics so that all the cards remain the same length.

Reply:
@abdulkdawson Is it fine pick the first two instructors and first two topics in the list? I don't know if the instructors are in any particular order

Reply:
Sorry I'm late on this, and yea, `course_home` is only used for the course home page and is a valid way to verify that the page is being rendered with the `course_home.html` layout template.